docs/doc-tools/README.md

```diff
@@ -2,49 +2,50 @@
 
 This document teaches you how to make documentation in PDF.
 
+## Step One: Merge
 
-
-## Step One: Merge 
 Use the provided script `merge-all.py` to merge all the markdown doc files into one.
 
 ## Step Two: Generate TOC
+
 Use pandoc to generated TOC (table of content) for the file you just merged. First off, you should make sure pandoc has installed on your machine. Open your terminal and run the command:
```

Comment:
Firstly, you should make sure that pandoc has been installed on your computer. Open your terminal and run the command: